### ISSUE TITLE
Docker becomes even slower is Parallels Desktop is started 

### ISSUE THREAD

**Issue by ilg-ul** [docker/for-mac#894]
### Expected behaviour
If enough memory is available, starting Parallels Desktop should not impact Docker performances

### Actual behaviour
Starting Parallels Desktop while Docker is running seems to make Docker even much slower than it is usually. 

### Information

  - Full output of the diagnostics from "Diagnose & Feedback" in the menu

Docker 1.12.3-beta29.2

```
OS X: version 10.12.1 (build: 16B2555)
logs: /tmp/D71F5326-0D23-4376-8A06-67DBD61AF38C/20161103-185130.tar.gz
[OK]     vmnetd
[OK]     dns
[OK]     driver.amd64-linux
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     db
[OK]     env
[OK]     virtualization kern.hv_support
[OK]     slirp
[OK]     osxfs
[OK]     moby-console
[OK]     logs
[OK]     docker-cli
[OK]     menubar
[OK]     disk
Failure: Could not upload diagnostic data to remote server (docker-diagnose exit code is 1)
```
Attached manually:
[20161103-185130.tar.gz](https://github.com/docker/for-mac/files/569625/20161103-185130.tar.gz)

### Steps to reproduce the behavior

  1. start a very long build (for example [GNU ARM Eclipse QEMU](http://gnuarmeclipse.github.io/qemu/build-procedure/) is notably long)
  2. start a Parallels virtual machine
  3. stop the Parallels VM

The curious thing is that even if Parallels is closed, the Docker speed remains low.

**Comment by ijc25** [comment]
I expect that Parallels likely installs kernel modules which interfere somehow with OSX's native Hypervisor.framework (much like older virtual box 4.x modules did). Even shutting down Parallels probably doesn't unload those modules, maybe it is possible to do it manually though?

Unfortunately I think it is unlikely we can address that within Docker for mac. Have you reported this to one or both of Parallels or Apple?

**Comment by ilg-ul** [comment]
> Have you reported this to Parallels or Apple

nope. 

given past experience with Apple, I guess that, since Docker is affected, they'll ask me to report the problem to... Docker :-(

**Comment by dsheets** [comment]
@ilg-ul do you still experience this behavior? Have you tried unloading the Parallels kernel modules?

**Comment by ilg-ul** [comment]
I planned to upgrade Parallels to the latest version and time some builds, but I've been quite busy.

I'll come with more data, when available.

**Comment by ilg-ul** [comment]
these days I did some tests with the OpenOCD build and the results are not conclusive. :-(

the build with the Parallels Desktop **not** started, took:

```
Script "/Users/ilg/Downloads/build-scripts.git/scripts/build-openocd.sh" completed at Tue Jan 24 21:39:35 EET 2017.
Duration: 58 minutes.

real	58m28.223s
user	2m40.798s
sys	0m53.390s
```

**Exactly** the same script, with Parallels running a Windows 8 x64 VM, took only slightly more:

```
Script "/Users/ilg/Downloads/build-scripts.git/scripts/build-openocd.sh" completed at Wed Jan 25 14:58:11 EET 2017.
Duration: 59 minutes.

real	59m2.521s
user	3m14.665s
sys	0m55.462s
```

I did not have time to test the QEMU builds, which, last time I ran, took several hours to complete.

on the other side, now I'm using Version 1.13.0-beta38 (15084), not 1.12 stable, as I did for the QEMU build. it is perfectly possible for the latest releases to have fixed some performance issues.

I suggest we close this issue for now. when I'll run the QEMU builds I'll check again, and, if necessary, I'll report again.
